sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -217,7 +217,7 @@ private AnalyzeSentimentResult convertToAnalyzeSentimentResult(DocumentSentiment
             options.getModelVersion(), options.isIncludeStatistics(), options.isIncludeOpinionMining(),
             getNonNullStringIndexType(options.getStringIndexType()),
             getNotNullContext(context).addData(AZ_TRACING_NAMESPACE_KEY, COGNITIVE_TRACING_NAMESPACE_VALUE))
-                   .doOnSubscribe(ignoredValue -> logger.info("A batch of documents - {}", documents.toString()))
+                   .doOnSubscribe(ignoredValue -> logger.info("A batch of documents - {}", documents))
```

Comment:
Logging 100s of documents at the `info` level seems like an overkill. We should consider removing this log or changing this to verbose or just logging the count of docs.

Reply:
@mssfang, @samvaity, any input on which path to take here?

Overall, this is something would should review across the entirety of the repository as this isn't the only location where we may be logging at a level which will cause logging output to be too verbose.

Reply:
>just logging the count of docs

I would think this should suffice.